mmv1/third_party/terraform/resources/resource_logging_project_sink.go

```diff
@@ -55,13 +62,36 @@ func resourceLoggingProjectSinkCreate(d *schema.ResourceData, meta interface{})
 	id, sink := expandResourceLoggingSink(d, "projects", project)
 	uniqueWriterIdentity := d.Get("unique_writer_identity").(bool)
 
-	_, err = config.NewLoggingClient(userAgent).Projects.Sinks.Create(id.parent(), sink).UniqueWriterIdentity(uniqueWriterIdentity).Do()
-	if err != nil {
-		return err
+	create := true
+	name := d.Get("name").(string)
+	for _, reserved := range ReservedLoggingProjectSinks {
+		if name == reserved {
+			if err := resourceLoggingProjectSinkRead(d, meta); err != nil {
+				if isGoogleApiErrorWithCode(err, 404) {
```

Comment:
Similar to the above comment, I don't think Read ever throws a 404 error. I believe `handleNotFoundError()` effectively swallows the 404, because it indicates that a resource does not exist and Terraform should try to create it.